src/templates/Facility.vue

```diff
@@ -61,6 +61,11 @@
                         :to="{ name: 'update-contact', params: { entityID: entity.id, contactID: entity.contacts[0].id }}"
                 >{{ entity.contacts[0].name }}</router-link>
               </p>
+              <p
+                v-if="entity.contacts[0].entityContacts !== null && entity.contacts[0].entityContacts.relationshipTitle"
```

Comment:
Nit: I think you can do just do `entity.contacts[0].entityContacts && entity.contacts[0].entityContacts.relationshipTitle` without the null comparison?

Reply:
Was repeating the pattern for the other fields. Dont' want an undefined error, but I presume it would happen anyway on the check in the same statement? I'll test that and refactor those sections.